README.md

```diff
@@ -60,15 +61,25 @@ follows:
 ``` ps1
 $env:PMHC_USERNAME='your_username_here'
 $env:PMHC_PASSWORD=python -c 'import getpass; print(getpass.getpass())'
+$env:PMHC_TOTP_SECRET=python -c 'import getpass; print(getpass.getpass("TOTP Secret: "))'
 ```
 
 In a Unix shell (Mac, Linux), you can do:
 
 ``` bash
 export PMHC_USERNAME='your_username_here'
 read -rs PMHC_PASSWORD && export PMHC_PASSWORD
+read -rs PMHC_TOTP_SECRET && export PMHC_TOTP_SECRET
 ```
 
+NOTE: `PMHC_TOTP_SECRET` is the unchanging base32-encoded TOTP secret,
+not the time-based six-digit code. You can likely find this secret in
+the 'advanced' section of your TOTP app. It will be a long string of
+upper-case letters and digits. The six-digit code will be automatically
```

Comment:
Can we also please suggest a couple of suitable Authenticator apps which can be used for this (e.g. Aegis Authenticator for Android), as this could be a little confusing for new users as most of the large Authenticator apps (e.g. Microsoft Authenticator) do not allow any way to access the underlying 32 character code.

The Android app store is also fraught with dodgy paid ads and fake apps when one searches 'authenticator app' so I think it would help guide the user on the right path!

Reply:
One can also obtain the 32 character string in clear text by scanning the QR code from PMHC website with a QR scanner - might be another option to suggest?

Reply:
Thanks. I'll add a note about known good QR code apps which support viewing the secret. I believe that there is a button on the same page as the QR code which will reveal the secret directly. However, it's impossible to get back to the QR code screen once you've set it up, so getting it from the setup page, either with the button or with a QR code scanner, will only work if you happen to be setting up MFA at the same time as setting up the pipeline.

Reply:
I actually looked for that option on the PMHC QR code screen but couldn't find it. Interesting how they don't let you ever get back to that page, probably good from a security point of view.

Reply:
Correction, the PMHC page _does_ list the 32 char secret if you do click following (thanks to @alextoland1 for these screenshots as I couldn't get back to this page!):

 - From the 2FA setup screen select **Google Authenticator or similar**
 - On the subsequent screen click **Trouble Scanning?**
 
 
![image](https://github.com/user-attachments/assets/7c8e26d1-53ee-4873-a8fb-fe48242c8f04)


Could we include this in the README.md as I think this could trip others up. 

Particularly if you say, scan the code using Microsoft Authenticator out of habit, then realise you want the 32 char string for pmhclib and want to try setup PMHC in another app e.g. Aegis Authenticator, you have no way to get the 32 character secret any more?

Reply:
Also tested these changes under Powershell (Python 3.12.2) and it worked well.

Reply:
Thanks Jono, I've added these improvements to the README.